app/modules/intelligence/tools/code_query_tools/git_push_tool.py

```
@@ -166,28 +256,54 @@ def _push_operation(repo):
+                    try:
+                        plain_url = (
+                            repo.git.remote("get-url", remote).strip().rstrip("/")
+                        )
+                        auth_url = self.repo_manager._build_authenticated_url(
+                            plain_url, auth_token, repo_name=repo_name
+                        )
+                        if auth_url:
+                            auth_url = auth_url.rstrip("/")
+                        if auth_url:
+                            repo.git.remote("set-url", remote, auth_url)
+                    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)